src/GitHub.Api/Application/ApplicationManagerBase.cs

```diff
@@ -42,6 +42,13 @@ protected void Initialize()
             Platform = new Platform(Environment);
 
             LogHelper.TracingEnabled = UserSettings.Get(Constants.TraceLoggingKey, false);
+
+            int webTimeout;
+            if (int.TryParse(UserSettings.Get(Constants.WebTimeoutKey), out webTimeout))
```

Comment:
settings will do the conversion for you - `UserSettings.Get<int>(Constants.WebTimeoutKey)`

Reply:
Fixed